tools/autograd/load_derivatives.py

```diff
@@ -781,6 +781,19 @@ def stride_expr(name: str) -> str:
                 ),
                 "expr": lambda name: f"{name}.has_value() ? c10::optional<IntArrayRef>({name}->sizes()) : c10::nullopt",
             },
+
+        ),
+        # replace self->sym_sizes() with self_sym_sizes_opt
+        (
+            r"{}->sym_sizes\(\)",
```

Comment:
Wait, we already replace sym_sizes() into self_sym_sizes two rules above :p 
One of the two needs to be removed.

Reply:
Ho that is subtle, my bad!